### PR TITLE
feat(ci,evm_transition_tool): Add EthereumJS

### PR DESCRIPTION
**Pull request by jochem-brouwer**
This PR adds an EthereumJS wrapper, which can be used to fill tests. Needs: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3603

### CODE REVIEW

**Review by marioevz — APPROVED** [review]
Looks good! I think we could merge now, and the only remaining thing before releasing would be the 2935 issue because it affects all Prague tests, even the ones unrelated to EIP-2935.

File: .github/actions/build-evm-client/ethjs/action.yml
Comment (jochem-brouwer) [reply .github/actions/build-evm-client/ethjs/action.yml:11]: Yes this is fine! However I will then wait marking this PR ready for review (and thus ready to merge) once we get t8ntool merged (which will take some time, since I need to do a massive cleanup of that PR)

File: .github/actions/build-evm-client/ethjs/action.yml
Comment (marioevz) [inline .github/actions/build-evm-client/ethjs/action.yml:28]:
```suggestion
      shell: bash
      run: |
```
Found during build in action: https://github.com/ethereum/execution-spec-tests/actions/runs/10492268620/job/29063368608

**Review by danceratopz — APPROVED** [review]
LGTM!

Just updated docs to reflect the addition (we should double check the links when https://github.com/ethereumjs/ethereumjs-monorepo/pull/3603 is merged).

File: .github/actions/build-evm-client/ethjs/action.yml
Comment (jochem-brouwer) [reply .github/actions/build-evm-client/ethjs/action.yml:11]: Have marked the PR ready for review, should be merged ASAP, will point to master.

**Review by marioevz — APPROVED** [review]
LGTM, thanks!

**Comment by jochem-brouwer** [comment]
Also got t8n merged on our side https://github.com/ethereumjs/ethereumjs-monorepo/pull/3603, so we are now ready-to go to fill some tests :smile: :rocket:

**Pull request merged**
